remme/rpc_api/_base.py

```diff
@@ -158,12 +159,13 @@ def register(self, ws, stream):
         # handle requests
         if msg.type == JsonRpcMsgTyp.REQUEST:
             self.logger.debug('msg gets handled as request')
+            method = msg.data['method']
 
             # check if method is available
-            if msg.data['method'] not in http_request.methods:
-                self.logger.debug('method %s is unknown or restricted',
-                                  msg.data['method'])
-                if msg.data['method'] in self.rpc_methods:
+            if method not in http_request.methods:
+                self.logger.debug('method %s is unknown or restricted', method)
```

Comment:
Use `f-strings` as on 178 code line.

Reply:
I made needed changes from format to `f-strings`. Thanks for commentary.